driver/ppm_events_public.h

```diff
@@ -273,6 +273,7 @@ or GPL2.txt for full copies of the license.
 #define PPM_SHUT_RD 0
 #define PPM_SHUT_WR 1
 #define PPM_SHUT_RDWR 2
+#define PPM_SHUT_UNKNOWN 0xffff
```

Comment:
We should add the string mapping here: https://github.com/falcosecurity/libs/blob/04d4cb7a2dd9307bcc3ccd334230d85101940cac/driver/flags_table.c#L202